docs/sdkx-react-native/notifications-android.mdx

```diff
@@ -0,0 +1,153 @@
+like iOS and Android. Notifications are useful to tell your users when
+you reply to an issue that they submitted.
+
+To know more about the FCM Push, refer:- [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging/)
+
+### Prerequisites {#gcm-prerequisites}
+
+Implement FCM push notifications in your app.
+
+For FCM, refer to the
+[Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) documentation.
+
+You can use any Firebase plugin from React Native Plugin. For eg - [React Native Firebase](https://rnfirebase.io/messaging/usage)
+
+### Configure Helpshift Agent Dashboard {#configure-helpshift-push-admin}
+
+To enable the Helpshift system to send push notifications to your users
+you will have to add an **Android** as a platform in your app _(if you have not added one already)_. And then click on the _push notifications_ option.
+
+<Image
+  src="/static/books/helpshiftx/create_app_with_android_platform.png"
+  width="full"
+/>{" "}
+
+Enter your FCM key credentials per app, via the
+**Settings page > Scroll down to App Settings the left pane > Select your App > Under Supported Platforms iOS/Android > Click Configure > Select Push
+Notification** section.
+
+<Image src="/static/books/helpshiftx/add_fcm_token.png" width="full" />{" "}
+
+For FCM users, the API key can be found at your
+[Firebase API Console](https://console.firebase.google.com/)
+
+<Image src="/static/books/android/fcm_console.png" width="full" />{" "}
+
+## Configure the Helpshift React Native Plugin SDK to handle notifications {#configure-helpshift-react-sdk}
+
+### Push notifications {#push-notifications}
+
+#### Configure FCM push notification plugins {#}
+
+After you have configured the push notifications in the Helpshift console, you will need to add additional setup in the project, too.
+
+First of all, you will need to set up the firebase push notification plugin where you will be getting the firebase token
+
+
+
+Then you will have to call the `Register Push Token` function once your application receives a push token from your provider:
+`registerPushToken(<token>)`
+
+For example with [React Native plugin](https://rnfirebase.io/messaging/usage), you can call the `const token = await requestNotificationPermission()` function to fetch the push registration token. Register this token with Helpshift SDK via the `Register Push Token` function.
+
+To handle the incoming push notifications on Android you will have to call `handlePush(<remoteMessage.data>)`.
+
+
+To check whether this notification is being sent from the Helpshift's push notification service, please check the `origin` field of the notification. If it is `"helpshift"`, the notification is a Helpshift notification.
+
+To handle notifications from the background we recommend calling `setBackgroundMessageHandler` inside the index.js file where your AppRegistry.registerComponent is called.
+```
+messaging().setBackgroundMessageHandler(async (remoteMessage) => {
+    init();
```

Comment:
init is an internal function that mentions the `install` call here not `init`. 
good to have an import for handlePush function gives more clarity.

Reply:
Also, give link https://rnfirebase.io/messaging/usage#background--quit-state-messages . Any changes to this plugin we don't own changes here

Reply:
@vikashelpshift please have a look.

Reply:
UPDATED with install